hq/app/logic/PublicBucketsDisplay.scala

```diff
@@ -27,6 +28,13 @@ object PublicBucketsDisplay {
     s"https://console.aws.amazon.com/s3/home?bucket=${URLEncoder.encode(bucket.bucketName, "utf-8")}"
   }
 
+  def isOnlyMissingEncryption(bucket: BucketDetail): Boolean = {
+    if (bucket.reportStatus.getOrElse(None) == Blue) {
```

Comment:
This looks like a comparison between an option and an int to me?

Reply:
I guess this function could be simplified to just return `bucket.reportStatus.getOrElse(None) == Blue && !bucket.isEncrypted`